File: Treinamento/C/Estrutura_sequencial/Questão08.c
#include <stdio.h>

int main() {
    double ganho_horario;
    int horas_dia;

    printf("Digite o seu ganho horário: ");
    scanf("%lf", &ganho_horario);

    printf("Digite sua jornada diária: ");
    scanf("%i", &horas_dia);

    printf("Salário: R$%lf reais\n", ganho_horario * horas_dia * 20);
}

File: Treinamento/Java/Estrutura_sequencial/Questão08.java
import java.util.Scanner;
public class Questão08 {
    public static void main(String[] args) {
        Scanner s;
        s = new Scanner(System.in);

        System.out.print("Digite o seu ganho horário: R$" );
        double g = s.nextDouble();

        System.out.print("Digite a jornada diária: ");
        int h = s.nextInt();

        System.out.println("Salário mensal: R$" + (g * h * 20) + " reais");
    }
}
